File: 01/prac16.js
function evenBigOddSmall(arr){

    for(let i =1;i<arr.length;i++){

        if((i+1)%2 ==0){
            if(arr[i] < arr[i-1]){
                [arr[i],arr[i-1]] = [arr[i-1],arr[i]]
            }
        }else {
            if(arr[i] > arr[i-1]){
                [arr[i],arr[i-1]] = [arr[i-1],arr[i]]
            }
        }

    }
return arr

}

let arr =[1,2,3,4,5,6]
console.log(evenBigOddSmall(arr))

File: javaProblems/prac16.java
package leetCode.javaProblems;

import java.util.LinkedHashSet;
import java.util.Set;

public class prac16 {
    
    public static void  removeDuplicate(int[]arr){

        Set <Integer> set = new LinkedHashSet<>();

        for(int a:arr){
            set.add(a);
        }

        System.out.println(set);
    }

public static void main(String[] args) {
    
    int[] arr = {1,2,3,4,4,5,6,5,3};
    removeDuplicate(arr);
}

}
